addons/reload/CfgVehicles.hpp

```diff
@@ -6,7 +6,7 @@ class CfgVehicles {
                 class GVAR(LinkBelt) {
                     displayName = CSTRING(LinkBelt);
                     distance = 2.0;
-                    condition = QUOTE([ARR_2(_player, _target)] call FUNC(canLinkBelt));
+                    condition = QUOTE(([ARR_2(_player, _target)] call FUNC(canLinkBelt) > 0));
```

Comment:
Yes. Woops little to fast maybe. 
Was thinking about it but it was to far of a leap 😛
Fnc_maxAmmiLinkBelt? Help me.